src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -36,6 +36,7 @@ class Grammar extends BaseGrammar
         'offset',
         'unions',
         'lock',
+        'between',
```

Comment:
Why is this needed?

Reply:
It was for my initial implementation, I'm removing it right now.